packages/playwright-core/src/dispatchers/playwrightDispatcher.ts

```diff
@@ -29,7 +29,8 @@ import { ElectronDispatcher } from './electronDispatcher';
 import { APIRequestContextDispatcher } from './networkDispatchers';
 import { SelectorsDispatcher } from './selectorsDispatcher';
 
-export class PlaywrightDispatcher extends Dispatcher<Playwright, channels.PlaywrightInitializer, channels.PlaywrightEvents> implements channels.PlaywrightChannel {
+export class PlaywrightDispatcher extends Dispatcher<Playwright, channels.PlaywrightChannel> implements channels.PlaywrightChannel {
+  _type_Playwright;
```

Comment:
Why not `_type_Playwright = true` like in other places?

Reply:
It does not compile :/